src/core/lib/transport/transport.h

```diff
@@ -71,9 +71,10 @@
 
 /** A comma separated list of supported transport protocols. If non-empty,
  allows the client and server to attempt to negotiate transport protocols.
- NOTE: This is an experimental feature. It is not fully implemented and is not
- currently functional.
- TODO(gtcooke94) - update with specific details when implementing. */
+ Transport protocol negotiation is currently during ALTS handshakes. If this
```

Comment:
Can we please revert the changes to this file? This is still an experimental feature, and we do not want to change this. We can provide full documentation when it is no longer experimental. :)